### ISSUE TITLE
fully_diluted_valuation in CoinMarkets should be Double

### ISSUE THREAD

**Issue by stolzem** [DrewCarlson/CoinGecko-Kotlin#34]
Class CoinMarkets has property fullyDilutedValuation declared as Long or Null.
Looks like it can be returned as double value as in the attached example.

```
@Serializable
data class CoinMarkets(
...
        @SerialName("fully_diluted_valuation")
        val fullyDilutedValuation: Long?,
...
)
```

https://api.coingecko.com/api/v3/coins/markets?vs_currency=btc&ids=tbtc&order=market_cap_desc&sparkline=false

```
  {
    "id": "tbtc",
    "symbol": "tbtc",
    "name": "tBTC",
    "image": "https://assets.coingecko.com/coins/images/11224/large/tBTC.png?1589620754",
    "current_price": 0.9955008,
    "market_cap": 819.466,
    "market_cap_rank": 729,
    **_"fully_diluted_valuation": 819.466,_**
    "total_volume": 105.879,
    "high_24h": 1.024948,
    "low_24h": 0.99549243,
    "price_change_24h": -0.025405981763,
    "price_change_percentage_24h": -2.48857,
    "market_cap_change_24h": -5.152366432831,
    "market_cap_change_percentage_24h": -0.62482,
    "circulating_supply": 823.17,
    "total_supply": 823.17,
    "max_supply": 823.17,
    "ath": 1.339358,
    "ath_change_percentage": -25.67334,
    "ath_date": "2020-09-23T17:59:54.391Z",
    "atl": 0.85864247,
    "atl_change_percentage": 15.93885,
    "atl_date": "2021-12-09T16:35:49.489Z",
    "roi": null,
    "last_updated": "2021-12-28T01:03:39.021Z"
  },

```

Parsing failes in this case. Type of fully_diluted_valuation should be changed to Double.
Using coingecko-jvm-0.3.0.jar.

**Comment by DrewCarlson** [comment]
Thanks, resolved in v0.3.1

**Comment by stolzem** [comment]
:+1:
